doc/integrator/install_application.rst

```diff
@@ -165,6 +167,23 @@ using a C compiler.
 Furthermore, some changes in the apache wsgi and mapserver configurations are
 required to make c2cgeoportal work on Windows.
 
+Also, between all the different command interfaces available on windows (cmd, 
+Cygwin, git mingw), only Windows default cmd interface handle paths correctly
+in all stage of the application setup.
+
+Command interface and environment variable
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Only use Windows default commande interface::
+
+    Start > Run... > cmd
+
+Cygwin and git mingw are not compatible. Powershell is untested.
+
+Complementarily you need to add all the ressource paths to your system PATH
+environment variable, for cygwin, git, node, apache, postgres, mapserver and
```

Comment:
Cool, thanks a lot for that ! I was not sure why all the apps you just removed were in the path, and I just wanted it to be clear (in my head...)

This PR is a really good thing (for Windows users...) and I am very grateful for it !